src/actions/noting/reset-note-barriers.js

```diff
@@ -0,0 +1,35 @@
+var isVFocus = require('../../utils/vfocus/is-vfocus');
+var isVText = require('../../utils/vfocus/is-vtext');
+var findAllNotes = require('../../utils/noting/find-all-notes');
+var createNoteBarrier = require('../../utils/create-note-barrier');
+var isNotWithinNote = require('../../utils/noting/is-not-within-note');
+var isNotEmpty = require('../../utils/vfocus/is-not-empty');
+var errorHandle = require('../../utils/error-handle');
+
+module.exports = function resetNoteBarriers(focus) {
```

Comment:
I think a comment about what the function does and why it exists would be good (I know it, but for the next person who looks at the code!).